packages/casting/test/fake-rpc-server.js

```diff
@@ -64,10 +64,19 @@ const fakeStatusResult = {
   },
 };
 
-export const startFakeServer = (t, fakeValues, marshaller = makeMarshal()) => {
+/** @typedef {Partial<import('ava').ExecutionContext<{cleanups: Array<() => void>}>> & {context}} fakeServerTestContext */
+/**
+ * @param {fakeServerTestContext} t
+ * @param {Array<{any}>} fakeValues
+ * @param {object} [options]
+ * @param {Marshaller} [options.marshaller]
+ * @param {number} [options.batchSize] count of stream-cell results per response, or 0/absent to return lone naked values
+ */
+export const startFakeServer = (t, fakeValues, options = {}) => {
   const { log = console.log } = t;
   lastPort += 1;
   const PORT = lastPort;
```

Comment:
Aside: this usually tells me we should listen on 0 and ask the kernel for the port we were assigned.